server/service/src/master_list/tests/query.rs

```diff
@@ -11,64 +12,52 @@ mod query {
             setup_all("test_master_list_filter", MockDataInserts::all()).await;
 
         let service_provider = ServiceProvider::new(connection_manager, "app_data");
-        let context = service_provider.basic_context().unwrap();
+        let context = service_provider
+            .context(mock_store_a().id, "".to_string())
+            .unwrap();
         let service = service_provider.master_list_service;
 
         let result = service
             .get_master_lists(
                 &context,
+                &context.store_id,
                 None,
                 Some(
                     MasterListFilter::new()
-                        .exists_for_name_id(EqualFilter::equal_to("id_master_list_filter_test")),
+                        .exists_for_name_id(EqualFilter::equal_to("name_store_a")),
                 ),
                 None,
             )
             .unwrap();
 
-        assert_eq!(result.count, 1);
-        assert_eq!(result.rows[0].id, "master_list_filter_test");
+        assert_eq!(result.count, 2);
+        assert_eq!(result.rows[0].id, "item_query_test1");
 
         let result = service
             .get_master_lists(
                 &context,
+                &context.store_id,
                 None,
-                Some(
-                    MasterListFilter::new()
-                        .exists_for_name(StringFilter::like("e_master_list_filter_te")),
-                ),
+                Some(MasterListFilter::new().exists_for_name(StringFilter::like("Store A"))),
                 None,
             )
             .unwrap();
 
-        let master_list_row = result.rows[0].clone();
-        assert_eq!(result.count, 1);
+        let master_list_row = result.rows[1].clone();
+        assert_eq!(result.count, 2);
         assert_eq!(master_list_row.id, "master_list_filter_test");
         assert_eq!(master_list_row.name, "name_master_list_filter_test");
 
         //Test filter on exists_for_store_id "store_a" finds something
         let result = service
             .get_master_lists(
                 &context,
+                &context.store_id,
                 None,
                 Some(MasterListFilter::new().exists_for_store_id(EqualFilter::equal_to("store_a"))),
                 None,
             )
             .unwrap();
         assert!(result.count >= 1);
-
-        //Test filter for non existent store finds nothing
```

Comment:
Test no longer needed since we're filtering out store_id by default... though do we still need the name_id filter now that we're filtering out by store_id?